### PR TITLE
[REVIEW] feat(Select): Keyboard events & ngModel capabilities

### PR DESCRIPTION
**Pull request by elenagiere**
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: angular/src/lib/list-item/list-item.component.ts [hunk 1]
```diff
@@ -102,6 +111,7 @@ export class ListItemComponent implements OnInit, AfterViewInit, AfterContentChe
       'cui-list-item' +
       `${(this.type && ` cui-list-item--${this.type}`) || ''}` +
       `${(this.active && ` active`) || ''}` +
+      `${(this.focus && ` focus`) || ''}` +
```
Comment (ArthusLiang) [inline angular/src/lib/list-item/list-item.component.ts]: I used @HostBinding before. It's a good example for using it here. Now I use :host under @Component. Maybe you can update the code.

File: angular/src/lib/select/select.component.ts [hunk 2]
```diff
@@ -1,21 +1,32 @@
-import {
-  Component, Input, HostBinding, ViewChild, AfterContentChecked, ContentChildren,
-  QueryList, ChangeDetectorRef, AfterContentInit, OnInit, NgZone, Output, EventEmitter
-} from '@angular/core';
-import { SelectionModel } from '@angular/cdk/collections';
+import {Component, Input, ViewChild, AfterContentChecked, ContentChildren, QueryList,
```
Comment (bfbiggs) [inline angular/src/lib/select/select.component.ts]: Can we make this multi-line?

File: angular/src/lib/list-item/list-item.component.ts (hunk 1 above)
Comment (elenagiere) [reply angular/src/lib/list-item/list-item.component.ts]: @ArthusLiang Yes, we want to change the way we do @HostBinding. This PR was about the SelectComponent in which I did change the style of HostBinding. If I change it in list-item, then I should change it in list and list-item-section. I think we should allocate a story to audit through the already existing components to make these adjustments.

Comment (elenagiere) [reply angular/src/lib/list-item/list-item.component.ts]: I went ahead and changed the Hostbinding of classNames for List and ListItem.

**Pull request merged**